js/data.js

```diff
@@ -6,3 +6,14 @@ let data = {
   editing: null,
   nextEntryId: 1,
 };
+
+function serialize() {
+  const dataModelJSON = JSON.stringify(data.entries);
```

Comment:
You should be serializing the whole `data` object, rather than just `data.entries`